maven-scm-providers/maven-scm-providers-svn/maven-scm-provider-svnexe/src/test/java/org/apache/maven/scm/provider/svn/svnexe/command/mkdir/SvnMkdirCommandTest.java

```diff
@@ -56,21 +59,23 @@ protected void setUp()
         messageFileString = "--file " + path + " --encoding UTF-8";
     }
 
-    protected void tearDown()
+    public void tearDown()
```

Comment:
Apparently, this still works (at least, no tests seem to be failing because this doesn't run) but wouldn't it be more nice to annotate with `@After` (in JUnit 4 style)?

Reply:
fixed